Jenkinsfile

```diff
@@ -31,19 +31,19 @@ pipeline {
                        execPattern: '**/target/jacoco.exec',
                        classPattern: '**/target/classes',
                        sourcePattern: '**/src/main/java'
-                recordIssues id: "jdk11", name: "Static Analysis jdk11", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle(), spotBugs(), pmdParser(), errorProne()]
+                recordIssues id: "jdk17", name: "Static Analysis jdk17", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle()]
               }
             }
           }
         }
 
-        stage("Build / Test - JDK17") {
+        stage("Build / Test - JDK11") {
           agent { node { label 'linux' } }
           steps {
             container( 'jetty-build' ) {
               timeout( time: 120, unit: 'MINUTES' ) {
-                mavenBuild( "jdk17", "clean install -Dspotbugs.skip=true -Djacoco.skip=true", "maven3")
-                recordIssues id: "jdk17", name: "Static Analysis jdk17", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle(), spotBugs(), pmdParser()]
+                mavenBuild( "jdk11", "clean install -Dspotbugs.skip=true -Djacoco.skip=true -Perrorprone", "maven3")
+                recordIssues id: "jdk11", name: "Static Analysis jdk11", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle(), errorProne()]
```

Comment:
`spotbugs()` and `pmdParser()` have been removed from both builds. On purpose?
If so, do we have sections/dependencies in the POM that we need to remove as well?

Reply:
pmd is no longer used in build, it shouldn't have been looked fro in the recordIssues().

spotbugs is only done for the first build, which is now jdk17.
However, spotbugs, like errorprone, does not work for jdk17 (yet).

We will fix spotbugs and errorprone in later PRs.